### PR TITLE
gpkg-sign add gpg configuration check.

### PR DESCRIPTION
**Pull request by syu-nya**
Closes: https://bugs.gentoo.org/869470

Signed-off-by: Sheng Yu <syu.os@protonmail.com>

### CODE REVIEW

File: bin/gpkg-sign [hunk 1]
```diff
@@ -16,6 +16,25 @@ def main(
     gpkg_file, keep_current_signature=False, allow_unsigned=False, skip_signed=False
 ):
     eout = portage.output.EOutput()
+
+    if (
+        "BINPKG_GPG_SIGNING_GPG_HOME" in portage.settings
+        and portage.settings["BINPKG_GPG_SIGNING_GPG_HOME"]
```
Comment (mgorny) [inline bin/gpkg-sign:22]:
```suggestion
        not portage.settings.get("BINPKG_GPG_SIGNING_GPG_HOME")
```

perhaps? And so on.

**Review by mgorny — APPROVED** [review]
LGTM

**Comment by mgorny** [comment]
(note that I haven't actually tested it, just eyeball-review)

**Comment by RinCat** [comment]
I will push a new update soon.

**Comment by syu-nya** [comment]
How did I trigger a github bug again. 😂

File: lib/_emerge/actions.py [hunk 2]
```diff
@@ -614,6 +614,28 @@ def action_build(
                         in trees[eroot]["root_config"].settings.features
                     )
                 ):
+                    if not trees[eroot]["root_config"].settings.get(
```
Comment (mgorny) [inline lib/_emerge/actions.py:617]: Well, the one above was rather short but this really asks for a:

```
for x in ("BINPKG_GPG_SIGNING_GPG_HOME", "BINPKG_GPG_SIGNING_KEY"):
    ...
```

Comment (syu-nya) [reply lib/_emerge/actions.py]: OK, changed.

**Review by mgorny — APPROVED** [review]
Again, without testing, but eyeball LGTM.

**Comment by thesamesam** [comment]
> Again, without testing, but eyeball LGTM.

Could you test it please, given you reported the bug?

**Pull request closed without merging**